apps/web/composables/useCategoryFilter/types.ts

```diff
@@ -0,0 +1,24 @@
+import { AgnosticCategoryTree } from '@plentymarkets/plentymarkets-sdk/packages/sdk/src/getters/agnostic.types';
+
+export interface GetFacetsFromURLResponse {
+  categorySlug?: string;
+  facets?: string;
+  itemsPerPage?: number;
+  page?: number;
+  sort?: string;
+  term?: string;
+  categoryId?: string;
+}
+
+export interface Filters {
+  [key: number]: string;
+}
+
+export interface UseCategoryFiltersResponse {
+  getFacetsFromURL(): GetFacetsFromURLResponse;
+  getCatLink(category: AgnosticCategoryTree): string;
```

Comment:
Use "CategoryTreeItem" instead of "AgnosticCategoryTree", we are away from the "AgnosticCategoryTree".